frontend/awx/interfaces/WorkflowJobTemplate.ts

```diff
@@ -4,7 +4,7 @@ import { SummaryFieldsByUser } from './summary-fields/summary-fields';
 export interface WorkflowJobTemplate
   extends Omit<SwaggerWorkflowJobTemplate, 'id' | 'name' | 'summary_fields' | 'job_type'> {
   id: number;
-  job_type: 'workflow_job_template';
+  job_type: 'workflow_job_template' | string;
```

Comment:
when is then ever not a 'workflow_job_template'?
with this change we lose the ability for typescript to use that field to determine the type of the resource.